warehouse/templates/pages/help.html

```diff
@@ -209,7 +209,7 @@ <h3 id="accessibility">{{ accessibility() }}</h3>
 
         <h3 id="uploading">{{ uploading() }}</h3>
         <p>
-          This feature was deprecated with the new version of PyPI – we instead recommend that you <a href="https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi">use twine to upload your project to PyPI</a>.
+          On legacy PyPI, it used to be possible for maintainers to upload releases to PyPI using a form in the web browser. This feature was deprecated with the new version of PyPI – we instead recommend that you <a href="https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi">use twine to upload your project to PyPI</a>.
```

Comment:
Kind of the same thought here about the term "legacy PyPI", this also might not make sense to users unfamiliar with the history. Maybe "On previous versions of PyPI"?